com.longbridgeapp.LongbridgePro.yaml

```diff
@@ -0,0 +1,81 @@
+app-id: com.longbridgeapp.LongbridgePro
+default-branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+command: longbridgepro
+
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=x11
+  - --talk-name=org.freedesktop.Notifications
+
+modules:
+  - name: longbridgepro
+    buildsystem: simple
+    build-commands:
+      - install -D longbridgepro.sh /app/bin/longbridgepro
+      - |
+        for s in 16 32 48 64 128 256 512; do
+          install -Dm644 longbridgepro_${s}.png ${FLATPAK_DEST}/share/icons/hicolor/${s}x${s}/apps/com.longbridgeapp.LongbridgePro.png
+        done
+      - install -Dm644 com.longbridgeapp.LongbridgePro.metainfo.xml -t /app/share/metainfo
+      - install -Dm644 com.longbridgeapp.LongbridgePro.desktop -t /app/share/applications
+      - install -D apply_extra -t /app/bin
+    sources:
+      - type: script
+        dest-filename: apply_extra
+        commands:
+          - |
+            bsdtar -Oxf longbridgepro.deb 'data.tar.xz' |
+              bsdtar -xf -
+          - mv opt/Longbridge\ Pro longbridgepro
+          - rm -r longbridgepro.deb opt usr
+      - type: file
+        path: com.longbridgeapp.LongbridgePro.desktop
+      - type: file
+        path: com.longbridgeapp.LongbridgePro.metainfo.xml
+      - type: file
+        path: longbridgepro.sh
+      - type: dir
```

Comment:
1. don't use `dir`
2. since the icons don't come from upstream, just keep the big one and discard everything else.